2018-04-02 13:19:22|ERROR|App.User.updateById|PDOException: Unknown column 'major' in 'field list' in D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php:261
Stack trace:
#0 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(409): NotORM_Result->query('UPDATE user SET...', Array)
#1 D:\Development\www\SchoolRushServer\src\app\Model\User.php(45): NotORM_Result->update(Array)
#2 D:\Development\www\SchoolRushServer\src\app\Api\User.php(248): App\Model\User->updateById('23', Array)
#3 [internal function]: App\Api\User->updateById()
#4 D:\Development\www\SchoolRushServer\vendor\phalapi\kernal\src\PhalApi.php(49): call_user_func(Array)
#5 D:\Development\www\SchoolRushServer\public\index.php(9): PhalApi\PhalApi->response()
#6 {main}
2018-04-02 14:15:34|ERROR|App.User.updateById|PDOException: Unknown column 'major' in 'field list' in D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php:261
Stack trace:
#0 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(409): NotORM_Result->query('UPDATE user SET...', Array)
#1 D:\Development\www\SchoolRushServer\src\app\Model\User.php(45): NotORM_Result->update(Array)
#2 D:\Development\www\SchoolRushServer\src\app\Api\User.php(248): App\Model\User->updateById('23', Array)
#3 [internal function]: App\Api\User->updateById()
#4 D:\Development\www\SchoolRushServer\vendor\phalapi\kernal\src\PhalApi.php(49): call_user_func(Array)
#5 D:\Development\www\SchoolRushServer\public\index.php(9): PhalApi\PhalApi->response()
#6 {main}
2018-04-02 17:00:12|ERROR|App.User.updateById|PDOException: Unknown column 'major' in 'field list' in D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php:261
Stack trace:
#0 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(409): NotORM_Result->query('UPDATE user SET...', Array)
#1 D:\Development\www\SchoolRushServer\src\app\Model\User.php(45): NotORM_Result->update(Array)
#2 D:\Development\www\SchoolRushServer\src\app\Api\User.php(255): App\Model\User->updateById(NULL, Array)
#3 [internal function]: App\Api\User->updateById()
#4 D:\Development\www\SchoolRushServer\vendor\phalapi\kernal\src\PhalApi.php(49): call_user_func(Array)
#5 D:\Development\www\SchoolRushServer\public\index.php(9): PhalApi\PhalApi->response()
#6 {main}
2018-04-02 17:11:34|ERROR|App.User.updateById|PDOException: Duplicate entry 'tete@qq.com' for key 'user_email_uindex' in D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php:261
Stack trace:
#0 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(409): NotORM_Result->query('UPDATE user SET...', Array)
#1 D:\Development\www\SchoolRushServer\src\app\Model\User.php(45): NotORM_Result->update(Array)
#2 D:\Development\www\SchoolRushServer\src\app\Api\User.php(269): App\Model\User->updateById('21', Array)
#3 [internal function]: App\Api\User->updateById()
#4 D:\Development\www\SchoolRushServer\vendor\phalapi\kernal\src\PhalApi.php(49): call_user_func(Array)
#5 D:\Development\www\SchoolRushServer\public\index.php(9): PhalApi\PhalApi->response()
#6 {main}
2018-04-02 17:12:34|ERROR|App.User.updateById|PDOException: Duplicate entry 'tete@qq.com' for key 'user_email_uindex' in D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php:261
Stack trace:
#0 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(409): NotORM_Result->query('UPDATE user SET...', Array)
#1 D:\Development\www\SchoolRushServer\src\app\Model\User.php(45): NotORM_Result->update(Array)
#2 D:\Development\www\SchoolRushServer\src\app\Api\User.php(270): App\Model\User->updateById('21', Array)
#3 [internal function]: App\Api\User->updateById()
#4 D:\Development\www\SchoolRushServer\vendor\phalapi\kernal\src\PhalApi.php(49): call_user_func(Array)
#5 D:\Development\www\SchoolRushServer\public\index.php(9): PhalApi\PhalApi->response()
#6 {main}
2018-04-02 17:14:53|ERROR|App.User.updateById|PDOException: Duplicate entry 'tete@qq.com' for key 'user_email_uindex' in D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php:261
Stack trace:
#0 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(409): NotORM_Result->query('UPDATE user SET...', Array)
#1 D:\Development\www\SchoolRushServer\src\app\Model\User.php(45): NotORM_Result->update(Array)
#2 D:\Development\www\SchoolRushServer\src\app\Api\User.php(270): App\Model\User->updateById('21', Array)
#3 [internal function]: App\Api\User->updateById()
#4 D:\Development\www\SchoolRushServer\vendor\phalapi\kernal\src\PhalApi.php(49): call_user_func(Array)
#5 D:\Development\www\SchoolRushServer\public\index.php(9): PhalApi\PhalApi->response()
#6 {main}
2018-04-02 17:20:05|ERROR|App.User.add|PDOException: Unknown column 'major' in 'field list' in D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php:261
Stack trace:
#0 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(338): NotORM_Result->query('INSERT INTO use...', Array)
#1 D:\Development\www\SchoolRushServer\vendor\phalapi\notorm\src\NotORM\Result.php(355): NotORM_Result->insert_multi(Array)
#2 D:\Development\www\SchoolRushServer\src\app\Model\User.php(32): NotORM_Result->insert(Array)
#3 D:\Development\www\SchoolRushServer\src\app\Api\User.php(184): App\Model\User->add(Array)
#4 [internal function]: App\Api\User->add()
#5 D:\Development\www\SchoolRushServer\vendor\phalapi\kernal\src\PhalApi.php(49): call_user_func(Array)
#6 D:\Development\www\SchoolRushServer\public\index.php(9): PhalApi\PhalApi->response()
#7 {main}
